### ISSUE TITLE
How do you use it?

### ISSUE THREAD

**Issue by adriel** [mifi/VideoGrader#16]
Hey, any tips on how to use this?
I've run the commands in https://github.com/mifi/VideoGrader#running (there were a few errors, but managed resolve them, I believe), now I get a white box.
I've tried dragging a .mov file into it, but it doesn't do anything.

On a Mac if that makes any difference.

Any tips?

PS, love the idea, sounds like this is exactly what I'm after. Doing trial and error colour grading in ffmpeg commands was driving me crazy.

<img width="864" alt="image" src="https://user-images.githubusercontent.com/154217/157385938-90b0ee0a-202a-4f04-a569-bb056b3e115a.png">
<img width="1236" alt="image" src="https://user-images.githubusercontent.com/154217/157386485-d6011861-4fe7-497b-9bd8-16cacf0498c8.png">
<img width="556" alt="image" src="https://user-images.githubusercontent.com/154217/157386529-9d8c89db-5cba-4a4f-b560-ec4e17443277.png">

**Comment by mifi** [comment]
hi. haven't really given this much love lately. maybe you can open developer tools and see if there is an error in the videograder window

**Comment by adriel** [comment]
Hey! Thanks for replying. TBH didn't expect one.

Here is the error I'm getting: 
<img width="632" alt="image" src="https://user-images.githubusercontent.com/154217/157596105-66957b54-ef28-469e-bd3a-752ae44ab78a.png">
<img width="722" alt="image" src="https://user-images.githubusercontent.com/154217/157596148-498243ef-7467-4cd4-b6d2-03605d2b52ab.png">

**Comment by mifi** [comment]
which version of electron are you running?

**Comment by adriel** [comment]
17.1.1 I believe
<img width="683" alt="image" src="https://user-images.githubusercontent.com/154217/157603893-1fab7fb9-8ed7-463c-89ee-0c0a890f24aa.png">

**Comment by mifi** [comment]
oh. you need to use electron v2. see package.json

unfortunately this project is really outdated and needs some love to get up to date

**Comment by adriel** [comment]
oooh, yeah, v17 is a tad newer.
I got a tonne of errors when I ran `yarn` on the existing lock file, me not really knowing what I was doing cleared the lock file and some other stuff which got it to "run" without errors, but then the electron app via the `npm` commands didn't work.

I just re-ran `yarn` using your lock file and while there are still a lot of errors (mostly related to `fsevents`), when running the `npm` commands, the electron app does load up and seems to work fine.

I'm happy to call it here. Thank you so much for the help (and tool!).

<img width="1070" alt="image" src="https://user-images.githubusercontent.com/154217/157608610-cefd2b1f-2b11-455d-8f10-3f89abd68650.png">
